packages/server/lambdas/database/client/index.ts

```diff
@@ -11,15 +13,21 @@ let databasePassword: string | null = null;
 
 export class DatabaseClient {
   public static async build(): Promise<PrismaClient> {
-    if (!databaseName || !databaseDomain || !databasePassword) {
-      [databaseName, databaseDomain, databasePassword] = await Promise.all([
-        getParameter(databaseNameId),
-        getParameter(databaseDomainId),
-        getSecuredStringParameter(databasePasswordId),
-      ]);
-    }
+    let DATABASE_URL: string;
+    const setLocalDatabaseConnection = environment === ENVIRONMENTS.local;
+    if (setLocalDatabaseConnection) {
+      DATABASE_URL = 'postgresql://root:root@host.docker.internal:5432/test_db?schema=public';
+    } else {
```

Comment:
We are not allowed to get ssm keys if locally? 
How about remove if-else for `setLocalDatabaseConnection` and replace with this?
``` 
let DATABASE_URL = setLocalDatabaseConnection ? "postgresql://root:root@host.docker.internal:5432/test_db?schema=public" : "postgresql://${databaseName}:${databasePassword}@${databaseDomain}:5432/${databaseName}?schema=public"
```

Reply:
If we're running locally, there's no need for SSM stuff.